routes/api/comments.js

```diff
@@ -6,42 +6,56 @@ const createError = require('http-errors');
 const router = express.Router();
 const { sequelize, CommentLike } = require("../../db/models");
 
-
-
-//Like a comment
+/* Like a comment */
 router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) => {
 
     if (!res.locals.authenticated || res.locals.user.id !== like.userId) {
+        /* Return error 401, if
+               user is not authenticated, or
+               user is not liker */
         return next(createError(401)); 
-    } else if (res.locals.authenticated ) {
-    
-    const likedComment = await CommentLike.create({
-            userId: res.locals.user.id,
-            commentId: req.params.id
-    })
-  }
-            res.status(201).json(likedComment);
-  }));
 
-
-//Unlike a comment
-router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
-    const like = CommentLike.findByPk(req.params.id);
-
-    if (!like) {res.status(404)};
+    } else if (res.locals.authenticated ) {
         
-    if (!res.locals.authenticated || res.locals.user.id !== like.userId) {
-        return next(createError(401)); 
-
-    } else if (res.locals.user.id === like.userId && res.locals.authenticated ) {
-             await CommentLike.destroy({where: {id:req.params.id}})
+        /* Create comment like */
+        const likedComment = await CommentLike.create({
+                userId: res.locals.user.id,
+                commentId: req.params.id
+            })
         }
-    
-                res.json({message: "Comment deleted"});
 
-}))
+        /* Set status 201, and
+           respond with json, likedComment */
+        res.status(201).json(likedComment);
+    }));
 
 
+/* Unlike a comment */
+router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
+
+        /* Initialize like object */
+        const like = CommentLike.findByPk(req.params.id);
 
+        /* Set status 404; 
+           there is no like object */
+        if (!like) {res.status(404)};
```

Comment:
`res.status` just sets the header of the response. Sending it using `res.statusCode(404)` is an option, but I think it's best we utilize error handlers here. We'll also need to make sure that the rest of the code does not attempt to execute, so be sure to return in the if statement.

To utilize error handlers, use next with an error similar to when `res.locals.authenticated` is `false`.

Reply:
Code has been changed to use next and use the error handler.